src/exchanges/gemini-client.js

```diff
@@ -229,6 +292,12 @@ class GeminiClient extends EventEmitter {
   _onMessage(remote_id, raw) {
     let msg = JSON.parse(raw);
     let subscription = this._subscriptions.get(remote_id);
+    if (!subscription) {
+      // if regular subscription isn't available, try the top_of_book special subscription
+      // which is used for ticker support
+      const subscriptionId = remote_id + this.topOfBookIndicator;
+      subscription = this._subscriptions.get(subscriptionId);
```

Comment:
we don't have any way of knowing in _onMessage if the message received was for a regular subscription or a "top of book" subscription. so if the regular one isn't found, just fallback to try to find the "top of book" version